src/operator-docs/transformation/windowToggle.ts

```diff
@@ -1,6 +1,81 @@
 import { OperatorDoc } from '../operator.model';
 
 export const windowToggle: OperatorDoc = {
-  'name': 'windowToggle',
-  'operatorType': 'transformation'
+  name: 'windowToggle',
+  operatorType: 'transformation',
+  signature:
+    'public windowToggle(openings: Observable<O>, closingSelector: function(value: O): Observable): Observable<Observable<T>>',
```

Comment:
In contrast with the old documentation, we don't want any "generics" in the method signature as per [#196](https://github.com/ReactiveX/rxjs-docs/pull/196#discussion_r155799478). So please remove the generics.

Reply:
Okay, will do it